soc/intel/intel_adsp/tools/cavstool.py

```diff
@@ -21,15 +21,18 @@
 HUGEPAGESZ = 2 * 1024 * 1024
 HUGEPAGE_FILE = "/dev/hugepages/cavs-fw-dma.tmp."
 
-# SRAM windows.  Each appears in a 128k region starting at 512k.
+# SRAM windows. Base and stride varies depending on ADSP version
 #
 # Window 0 is the FW_STATUS area, and 4k after that the IPC "outbox"
 # Window 1 is the IPC "inbox" (host-writable memory, just 384 bytes currently)
-# Window 2 is unused by this script
+# Window 2 is debug, divided into multiple slots, unused by this script
 # Window 3 is winstream-formatted log output
-OUTBOX_OFFSET    = (512 + (0 * 128)) * 1024 + 4096
-INBOX_OFFSET     = (512 + (1 * 128)) * 1024
-WINSTREAM_OFFSET = (512 + (3 * 128)) * 1024
+
+WINDOW_BASE = 0x80000
+WINDOW_STRIDE = 0x20000
+
+WINDOW_BASE_ACE = 0x180000
+WINDOW_STRIDE_ACE = 0x8000
```

Comment:
Out of scope, but are ACE windows really that much smaller?  Only 32k?  128k total for the whole device?  That seems frustratingly low, actually.  I have occasional fantasies about moving "audio server" functionality down into the DSP and doing client stream mixing there, etc...  That gets cramped without a lot of mappable memory.

Reply:
That's indeed the case, the windows are configurable on DSP, but the interface exposed to host has a fixed stride of 32k.